src/effects.cpp

```diff
@@ -1,541 +1,660 @@
+#include "effects/matrix/PatternClock.h"        // No matrix dependencies
+
+#if ENABLE_AUDIO
+    #include "effects/matrix/spectrumeffects.h" // Musis spectrum effects
+    #include "effects/strip/musiceffect.h"      // Music based effects
+#endif
+
+#if FAN_SIZE
+    #include "effects/strip/faneffects.h"       // Fan-based effects
+#endif
+
+//
+// Externals
+//
+
+#if USE_HUB75
+    #include "ledmatrixgfx.h"
+    #include "effects/matrix/PatternPongClock.h"
+    #include "effects/matrix/PatternMandala.h"
+    // These effects require LEDMatrixGFX::getPolarMap()
+    #include "effects/matrix/PatternSMHypnosis.h"
+    #include "effects/matrix/PatternSMRainbowTunnel.h"
+    #include "effects/matrix/PatternSMRadialWave.h"
+    #include "effects/matrix/PatternSMRadialFire.h"
+
+
+    #if ENABLE_WIFI
+        #include "effects/matrix/PatternSubscribers.h"
+        #include "effects/matrix/PatternWeather.h"
+        #include "effects/matrix/PatternStocks.h"
+    #endif
+
+    #if USE_NOISE
+        #include "effects/matrix/PatternNoiseSmearing.h"
+        #include "effects/matrix/PatternSMSmoke.h"
+    #endif
+
+#endif
+
+#if USE_HUB75 || USE_MATRIX
+
+    #include "effects/matrix/PatternAnimatedGIF.h"
+    #include "effects/matrix/PatternSMStarDeep.h"
+    #include "effects/matrix/PatternSMAmberRain.h"
+    #include "effects/matrix/PatternSMBlurringColors.h"
+    #include "effects/matrix/PatternSMFire2021.h"
+    #include "effects/matrix/PatternSMNoise.h"
+    #include "effects/matrix/PatternSMPicasso3in1.h"
+    #include "effects/matrix/PatternSMSpiroPulse.h"
+    #include "effects/matrix/PatternSMTwister.h"
+    #include "effects/matrix/PatternSMMetaBalls.h"
+    #include "effects/matrix/PatternSMHolidayLights.h"
+    #include "effects/matrix/PatternSMGamma.h"
+    #include "effects/matrix/PatternSMFlowFields.h"
+    #include "effects/matrix/PatternSMSupernova.h"
+    #include "effects/matrix/PatternSMWalkingMachine.h"
+    #include "effects/matrix/PatternPongClock.h"
+    #include "effects/matrix/PatternMandala.h"
+    #include "effects/matrix/PatternQR.h"
+    #include "effects/matrix/PatternSM2DDPR.h"
+    #include "effects/matrix/PatternSMStrobeDiffusion.h"
+    #include "effects/matrix/PatternSerendipity.h"
+    #include "effects/matrix/PatternSwirl.h"
+    #include "effects/matrix/PatternPulse.h"
+    #include "effects/matrix/PatternWave.h"
+    #include "effects/matrix/PatternMaze.h"
+    #include "effects/matrix/PatternLife.h"
+    #include "effects/matrix/PatternSpiro.h"
+    #include "effects/matrix/PatternCube.h"
+    #include "effects/matrix/PatternCircuit.h"
+    #include "effects/matrix/PatternAlienText.h"
+    #include "effects/matrix/PatternRadar.h"
+    #include "effects/matrix/PatternBounce.h"
+    #include "effects/matrix/PatternSpin.h"
+    #include "effects/matrix/PatternMisc.h"
+#endif
+
+
+#ifdef USE_WS281X
+    #include "ledstripgfx.h"
+#endif
+
+// Global effect set version
+
+#define EFFECT_SET_VERSION 6
+
```

Comment:
With #761 in, this really, really, has to disappear from our codebase and awareness. :)

```suggestion
```